corehq/apps/sms/views.py

```diff
@@ -1945,10 +1953,14 @@ def page_context(self):
         from corehq.messaging.smsbackends.turn.models import SQLTurnWhatsAppBackend
         from corehq.messaging.smsbackends.infobip.models import InfobipBackend
 
-        turn_backend = SQLTurnWhatsAppBackend.active_objects.filter(
-            domain=self.domain,
-            hq_api_id=SQLTurnWhatsAppBackend.get_api_id()
-        )
+        if TURN_IO_BACKEND.enabled(self.domain):
+            turn_backend = SQLTurnWhatsAppBackend.active_objects.filter(
+                domain=self.domain,
+                hq_api_id=SQLTurnWhatsAppBackend.get_api_id()
+            )
+        else:
+            turn_backend = SQLTurnWhatsAppBackend.objects.none()
```

Comment:
Just return an empty QuerySet so that it would be considered as if user does not have a Turn.io BE.